docs/developer/guidelines/pr.md

```diff
@@ -85,16 +97,8 @@ Different guidelines apply depending on which repo you are contributing to.
 === "integrations-core"
 
     If you are contributing to [integrations-core](https://github.com/DataDog/integrations-core) all you need to do is use the [`release changelog new`](../ddev/cli.md#ddev-release-changelog-new) command.
-    It will take care of all the formalities.
-    Add the `changelog/no-changelog` label if you don't need a changelog.
-
-## Separation of concerns
+    It will add files in `changelog.d` folder inside the integrations that you have modified.
+    Commit these files and push them to your PR.
+    
+    If you decide that you do not need a changelog because the change you made won't be shipped with the agent, add the `changelog/no-changelog` label to the PR.
```

Comment:
```suggestion
    If you decide that you do not need a changelog because the change you made won't be shipped with the Agent, add the `changelog/no-changelog` label to the PR.
```